src/js/components/PageHeader.js

```diff
@@ -37,34 +52,51 @@ class PageHeader extends React.Component {
   }
 
   render() {
-    let props = this.props;
+    let {
+      children,
+      className,
+      dividerClassName,
+      mediaWrapperClassName
+    } = this.props;
+
+    let classes = classNames(
+      'container container-fluid container-pod container-pod-short',
+      'flush flush-top',
+      className
+    );
+
+    let dividerClasses = classNames(
+      'container-pod container-pod-short flush-top',
+      'container-pod-divider-bottom container-pod-divider-bottom-align-right',
+      'container-pod-divider-inverse',
+      dividerClassName
+    );
+
+    let mediaWrapperClasses = classNames(
+      'media-object-spacing-wrapper media-object-spacing-narrow',
+      'media-object-offset',
+      mediaWrapperClassName
```

Comment:
all of these are able to be objects now, in order to be able to remove the previous classes. you'd also need to change the `PropTypes`